frontend/src/boot/i18n.ts

```diff
@@ -0,0 +1,17 @@
+import { boot } from 'quasar/wrappers';
+import messages from 'src/i18n';
+import Vue from 'vue';
+import VueI18n from 'vue-i18n';
+
+Vue.use(VueI18n);
```

Comment:
Following along with @davidlaprade's suggestion, we should be able to use a snippet like this:

```ts
let locale = 'en';
const params = new URLSearchParams(window.location.search);
if (params.has('locale')) {
  locale = params.get('locale')!;
}
```

To allow locale to be overriden via a query param

Reply:
Here's a reference for how we parse query params in another part of the codebase. Ben's snippet seems correct, just figured I'd share: https://github.com/ScopeLift/umbra-protocol/blob/08130c0f962a7cbd496a2a9fc5d79833bcbc0c46/frontend/src/utils/payment-links.ts#L77